src/extension.ts

```diff
@@ -220,6 +220,15 @@ function initializeIconPaths(context: vscode.ExtensionContext) {
     ext.iconPaths.dark.registry = context.asAbsolutePath('resources/dark/registry.svg')
     ext.iconPaths.light.registry = context.asAbsolutePath('resources/light/registry.svg')
 
+    ext.iconPaths.dark.s3.bucket = context.asAbsolutePath('resources/dark/s3/bucket.svg')
+    ext.iconPaths.light.s3.bucket = context.asAbsolutePath('resources/light/s3/bucket.svg')
+
+    ext.iconPaths.dark.s3.folder = context.asAbsolutePath('third-party/resources/from-vscode/dark/folder.svg')
+    ext.iconPaths.light.s3.folder = context.asAbsolutePath('third-party/resources/from-vscode/light/folder.svg')
+
+    ext.iconPaths.dark.s3.file = context.asAbsolutePath('third-party/resources/from-vscode/dark/document.svg')
+    ext.iconPaths.light.s3.file = context.asAbsolutePath('third-party/resources/from-vscode/light/document.svg')
```

Comment:
Now that we're pinned to VS Code v1.42.0, could we try codicons instead of importing new SVGs?

https://code.visualstudio.com/updates/v1_42#_codicons-for-reuse

Reply:
Good idea. Matches the icon set.

![image](https://user-images.githubusercontent.com/1410453/82971773-580f7c80-9f88-11ea-9b58-21310ef0ad67.png)